protos/dcs/mission/v0/mission.proto

```diff
@@ -392,6 +406,7 @@ message StreamEventsResponse {
     // 8192 to give DCS plenty of space for new built-in events.
     ConnectEvent connect = 8192;
     DisconnectEvent disconnect = 8193;
+    PlayerSentChatEvent player_sent_chat = 8194;
```

Comment:
I think https://github.com/DCS-gRPC/rust-server/pull/89/commits/de2be5b7ab35969973fbc4d10163478aa7848c6b is missing the following change:

```suggestion
    PlayerSendChatEvent player_send_chat = 8194;
```

Reply:
Thanks. Addressed in https://github.com/DCS-gRPC/rust-server/pull/89/commits/bc90a094a394f68db7b6b327dffa23134b86d96d